arviz/plots/distplot.py

```diff
@@ -83,6 +84,78 @@ def plot_dist(
     -------
     ax : matplotlib axes
     """
+    if backend is None or backend.lower() in ("mpl", "matplotlib"):
+        ax = _plot_dist_mpl(
```

Comment:
Being picky here, and maybe commenting too soon. I bet a lot of these keyword arguments could be constructed in a dict as theyre the same between both boken and mpl and just passed onto bokeh or mpl